modules/contacts/contacts.service.js

```diff
@@ -0,0 +1,67 @@
+const Contact = require('./contacts.model');
+const {
+  CONTACT_NOT_FOUND,
+  CONTACT_ALREADY_EXIST,
+} = require('../../error-messages/contacts.messages');
+
+class ContactsService {
+  async getContacts() {
+    return await Contact.find();
+  }
+
+  async getContactById(id) {
+    const contact = await Contact.findById(id);
+
+    if (contact) {
+      return contact;
+    }
+
+    throw new Error(CONTACT_NOT_FOUND);
```

Comment:
please reverse it 
if (!contact) {
    throw new Error(CONTACT_NOT_FOUND); 
    }
 return contact;